other/web-js/client/src/client-api.js

```diff
@@ -0,0 +1,212 @@
+import jwtDecode from "jwt-decode";
+
+/**
+ * Connects to a MongoDB Atlas App Services App's client API over HTTPS
+ */
+export class ClientApi {
+  static constructBaseUrl(deployment) {
+    const { deployment_model, cloud, region } = deployment;
+    if (deployment_model === "GLOBAL") {
+      return `https://realm.mongodb.com/api/client/v2.0/`;
+    }
+    if (!cloud || !region) {
+      throw new Error(
+        `Must specify a cloud provider and region for LOCAL Apps. e.g. { "cloud": "aws", "region": "us-east-1" }`
+      );
+    }
+    return `https://${region}.${cloud}.realm.mongodb.com/api/client/v2.0/`;
+  }
```

Comment:
why are we constructing the URL rather than just passing the string?

Reply:
Don't have access to the strings in realm.json yet. Will use them directly when we can!